content/blog/go-support-pulumi-2-0/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Welcoming Go to the Pulumi Family"
+date: 2020-05-06T13:10:24-05:00
+meta_desc: "Go is fully supported in Pulumi 2.0"
+meta_image: meta.png
+authors:
+    - evan-boyle
+tags:
+    - Go
+---
+
+We use Go to build Pulumi, making everyone on our engineering team a Gopher. Our open-source pulumi engine and SaaS back is written in Go, which has quickly become "the language of cloud." [Pulumi 2.0](https://www.pulumi.com/blog/pulumi-2-0/) affirms our commitment to Go by welcoming it to the family of supported infrastructure as code languages.
+
+<!--more-->
+
+## What is Pulumi?
+
+Pulumi's approach to modern Infrastructure as Code summarized in four points.
+
+- Programming Languages. Offer the best languages and tools to declare infrastructure and gain best-in-class productivity and engineering capabilities, leverage world-class language and tooling innovation, and connect to existing communities and ecosystems.
+
+- Multi-Cloud. Give operators the tools to build consistent workflows across any cloud(s). Whether public, private, or hybrid — you can use the full breadth and depth of any cloud services.
+
+- Modern Application Architectures. Pulumi fully supports legacy workloads, but it is designed for the world of modern containers, serverless, and cloud-native architectures. Build the architecture that works for you by mixing and matching infrastructure components.
+
+- Cloud Engineering Teams. Pulumi is for the entire team; everyone ranging from developers, infrastructure and operations engineers, and security engineers can use Pulumi.  Together, they can tackle cloud solutions together without technology or team silos.
+
+## Giving you superpowers
+
+Pulumi gives you the productivity superpowers of real programming languages. But what can you do with them?
+
+### Serverless
+
+Pulumi has first-class support for serverless workloads, allowing your application and infrastructure code to live side by side (and both be written in Go if you want!). Easily deploy [a lambda running a Go handler on AWS]([https://github.com/pulumi/examples/tree/master/aws-go-lambda):
+
+```go
+		args := &lambda.FunctionArgs{
+			Handler: pulumi.String("handler"),
+			Role:    role.Arn,
+			Runtime: pulumi.String("go1.x"),
+			Code:    pulumi.NewFileArchive("./handler/handler.zip"),
+		}
+
+		// Create the lambda using the args.
+		function, err := lambda.NewFunction(
+			ctx,
+			"basicLambda",
+			args,
+			pulumi.DependsOn([]pulumi.Resource{logPolicy}),
+		)
+		if err != nil {
+			return err
+		}
+
+		// Export the lambda ARN.
+		ctx.Export("lambda", function.Arn)
+
```

Comment:
Does this indentation render well in the site?   We may need to undent all of these.

Reply:
example of code rendering 

![Screen Shot 2020-05-13 at 5 51 04 PM](https://user-images.githubusercontent.com/638672/81873960-71550980-9542-11ea-9f9d-49b537159f1e.png)